apidescription.xml

```diff
@@ -1068,7 +1068,126 @@
+      </param>
+      <method id="commerce.shopping.cart.select.payments" name="POST" apigee:displayName="Select Payments">
+        <apigee:tags>
+          <apigee:tag primary="true">Commerce API</apigee:tag>
+          <apigee:tag>Commerce</apigee:tag>
+        </apigee:tags>
+        <apigee:authentication required="true"/>
+        <apigee:example url="commerce/v2/shopping/carts/4d1315c9-9220-48b3-9882-a295db25bfe3.jphx1/payments.json"/>
+        <doc apigee:url="http://developer.ticketmaster.com/products-and-docs/apis/commerce/#select-payments">Select Payments!
+        </doc>
+        <request>
+          <param name="TMPS-Correlation-Id" required="false" type="xsd:string" style="header">
+            <doc>
+              The TMPS-Correlation-Id header will be propagate to all our system. In order to track down issue with your request. You should provide it and it should be unique for each request.
+            </doc>
+          </param>
+          <param name="Content-Type" required="true" type="xsd:string" style="header" default="application/json">
+          </param>
+          <representation>
+            <apigee:payload required="true">
+              <doc apigee:url="https://developer.ticketmaster.com">
+                The payload should contain a valid payment in json format.
+              </doc>
+              <apigee:content>
+                <![CDATA[
+                       {
+                          "pollingCallbackUrl" : "http://your.server:8080/your-webhook-uri/",
+                          "payments" : [ {
+                             "type": "wallet",
+                             "amount" : {
+                                "amount" : "100.00",
+                                "currency" : "USD"
+                             },
+                             "token" : "wallet-token-id",
+                             "cvv" : "1234",
+                             "selectedItems" : [ {
+                                "reservations" : [ "reservation-id-1", "reservation-id-2" ]
+                             } ]
+                          } ]
+                       }
+                    ]]>
+              </apigee:content>
+            </apigee:payload>
+          </representation>
+        </request>
+      </method>
+    </resource>
 
+    <!-- Add Delivery -->
+    <resource path="commerce/{version}/shopping/carts/{cartId}/deliveries.{format}">
+      <param name="version" required="true" type="xsd:string" style="template" default="v2">
+        <doc>The API Version</doc>
+      </param>
+      <param name="cartId" required="true" type="xsd:string" style="template" default="">
+        <doc>The Cart Id</doc>
+      </param>
+      <param name="format" required="true" type="xsd:string" style="template" default="json">
+        <doc>API Response Format</doc>
+      </param>
+      <method id="commerce.shopping.cart.add.delivery" name="POST" apigee:displayName="Select Deliveries">
```

Comment:
I also understood that there was a time when apigee didn't support the PATCH operation.